SOLIDITY_STYLE_GUIDE.md

```diff
@@ -1378,6 +1379,48 @@ function getStorageRoot(uint256 _blockHeight)
 }
 ```
 
+### Structs and Maps
+
+Document structs above the struct and struct fields above the respective field.
+
+`Good`
+
+```solidity
+/** A Bank has a single owner. You need to create one bank per person. */
```

Comment:
Let's also add multiline documentation example in `Good` sections.

Reply:
Done, see below. What do you think about the blank lines between the fields if they have a comment?